cli/Dockerfile-bin

```diff
@@ -1,15 +1,16 @@
 ## compile binaries
 FROM gcr.io/runconduit/go-deps:5a44df7f as golang
 ARG CONDUIT_VERSION
+ARG PLATFORMS="darwin linux windows"
```

Comment:
Did you intend to change `bin/docker-build-cli-bin` so that we can override `PLATFORMS` so that only one platform is built? Otherwise, I don't understand why the changes to this file are useful as-is.

Reply:
oh thanks, I must have removed this by mistake :(